### PR TITLE
HMIS data quality

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/client.rb [hunk 1]
```diff
@@ -408,6 +459,13 @@ def self.sections
           title: 'Overlapping Night-by-Night ES enrollments with other ES, SH, and TH',
           description: 'Client\'s receiving more than two overlapping ES NbN services are included.',
           required_for: 'All',
+          detail_columns: [
+            :destination_client_id,
+            :first_name,
+            :last_name,
+            :reporting_age,
+            :overlapping_post_move_in,
```
Comment (awisema) [inline drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/client.rb]:
```suggestion
            :overlapping_nbn,
```

File: drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/client.rb [hunk 2]
```diff
@@ -417,6 +475,13 @@ def self.sections
           title: 'Overlapping Homeless Service After Move-in in PH',
           description: 'Client\'s receiving more than two overlapping homeless nights are included.',
           required_for: 'Adults',
+          detail_columns: [
+            :destination_client_id,
+            :first_name,
+            :last_name,
+            :reporting_age,
+            :overlapping_post_move_in,
```
Comment (awisema) [inline drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/client.rb]: Not sure this is right based on the name, but it matches the description on line 45...
```suggestion
            :overlapping_pre_move_in,
```

File: drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb [hunk 3]
```diff
@@ -360,6 +372,18 @@ def self.sections # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexit
           title: 'Relationship to Head of Household',
           description: 'Relashionship to head of household is an invalid value',
```
Comment (awisema) [inline drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb]:
```suggestion
          description: 'Relationship to head of household is an invalid value',
```

File: drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb [hunk 4]
```diff
@@ -482,6 +646,19 @@ def self.sections # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexit
           title: 'Possible Missed Exit - ES, No Service in 90 Days or More',
           description: 'There is an expectation that clients will not stay indefinitely in emergency shelter, these clients have been in shelter more than 90 days',
```
Comment (awisema) [inline drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb:647]: Should these say something like
```suggestion
          description: 'There is an expectation that clients will not stay indefinitely in emergency shelter, these clients have not received service in more than 90 days',
```
While the assertion is true, the comparison is against `days_since_last_service` not `lot`

**Review by awisema — CHANGES_REQUESTED** [review]

File: drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb (hunk 4 above)
Comment (eanders) [reply drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb:647]: nope, those 3 are about not staying in shelter too long, "Possible Missed Exit - ES, No Service in 90 Days or More" etc. cover the not receiving service.

**Pull request merged**